packages/admin-next/dashboard/src/routes/discounts/details/loader.ts

```diff
@@ -0,0 +1,30 @@
+import { AdminDiscountsRes } from "@medusajs/medusa"
+import { Response } from "@medusajs/medusa-js"
+import { adminDiscountKeys } from "medusa-react"
+import { LoaderFunctionArgs } from "react-router-dom"
+
+import { medusa, queryClient } from "../../../lib/medusa"
+
+const discountDetailQuery = (id: string) => ({
+  queryKey: adminDiscountKeys.detail(id),
+  queryFn: async () =>
+    medusa.admin.discounts.retrieve(id, {
+      expand:
+        "regions," +
+        "rule.conditions.products," +
+        "rule.conditions.product_types," +
+        "rule.conditions.product_tags," +
+        "rule.conditions.product_collections," +
+        "rule.conditions.customer_groups",
+    }),
```

Comment:
suggestion: extract this to a const and also pass it to useAdminDiscount in `discount-detail.tsx` otherwise the expand won't be part of the refetch that occurs when the data becomes stale after a couple of minutes